protocols/migration/participants.py

```diff
@@ -583,3 +584,158 @@ def migrate_tumour_sample(self, tumour_sample):
+            self.old_model.LifeStatus.DECEASED: self.new_model.LifeStatus.deceased,
+            self.old_model.LifeStatus.UNBORN: self.new_model.LifeStatus.unborn,
+            self.old_model.LifeStatus.STILLBORN: self.new_model.LifeStatus.stillborn,
+            self.old_model.LifeStatus.MISCARRIAGE: self.new_model.LifeStatus.miscarriage,
+        }
+        return status_map.get(old_status)
+
+    def migrate_adopted_status(self, old_status):
+        status_map = {
+            self.old_model.AdoptedStatus.notadopted: self.new_model.AdoptedStatus.not_adopted,
+            self.old_model.AdoptedStatus.adoptedin: self.new_model.AdoptedStatus.adoptedin,
+            self.old_model.AdoptedStatus.adoptedout: self.new_model.AdoptedStatus.adoptedout,
+        }
+        return status_map.get(old_status)
+
+    def migrate_person_karyotypic_sex(self, old_pks):
+        pks_map = {
+            self.old_model.PersonKaryotipicSex.UNKNOWN: self.new_model.PersonKaryotipicSex.unknown,
+            self.old_model.PersonKaryotipicSex.XX: self.new_model.PersonKaryotipicSex.XX,
+            self.old_model.PersonKaryotipicSex.XY: self.new_model.PersonKaryotipicSex.XY,
+            self.old_model.PersonKaryotipicSex.XO: self.new_model.PersonKaryotipicSex.XO,
+            self.old_model.PersonKaryotipicSex.XXY: self.new_model.PersonKaryotipicSex.XXY,
+            self.old_model.PersonKaryotipicSex.XXX: self.new_model.PersonKaryotipicSex.XXX,
+            self.old_model.PersonKaryotipicSex.XXYY: self.new_model.PersonKaryotipicSex.XXYY,
+            self.old_model.PersonKaryotipicSex.XXXY: self.new_model.PersonKaryotipicSex.XXXY,
+            self.old_model.PersonKaryotipicSex.XXXX: self.new_model.PersonKaryotipicSex.XXXX,
+            self.old_model.PersonKaryotipicSex.XYY: self.new_model.PersonKaryotipicSex.XYY,
+            self.old_model.PersonKaryotipicSex.OTHER: self.new_model.PersonKaryotipicSex.other,
+        }
+        return pks_map.get(old_pks)
+
+    def migrate_sex(self, old_sex):
+        sex_map = {
+            self.old_model.Sex.MALE: self.new_model.Sex.male,
+            self.old_model.Sex.FEMALE: self.new_model.Sex.female,
+            self.old_model.Sex.UNKNOWN: self.new_model.Sex.unknown,
+        }
+        return sex_map.get(old_sex, self.new_model.Sex.undetermined)
+
+    def migrate_analysis_panels(self, old_panels):
+        return None if old_panels is None else [self.migrate_analysis_panel(old_panel=old_panel) for old_panel in
+                                                old_panels]
+
+    def migrate_analysis_panel(self, old_panel):
+        new_panel = self.convert_class(self.new_model.AnalysisPanel, old_panel)
+        new_panel.review_outcome = old_panel.reviewOutcome
+        new_panel.multiple_genetic_origins = old_panel.multipleGeneticOrigins
+        return self.validate_object(object_to_validate=new_panel, object_type=self.new_model.AnalysisPanel)
+
+    def migrate_files(self, old_files):
```

Comment:
This whole chunk related to `File` migration was copied over from Interpretation Requests models and we don't needed it here